storage/src/main/java/org/apache/kafka/storage/internals/log/OffsetsOutOfOrderException.java

```diff
@@ -14,16 +14,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-package kafka.common
+package org.apache.kafka.storage.internals.log;
 
 /**
- * Indicates the follower or the future replica received records from the leader (or current
- * replica) with first offset less than expected next offset. 
- * @param firstOffset The first offset of the records to append
- * @param lastOffset  The last offset of the records to append
+ * Indicates the follower received records with non-monotonically increasing offsets
  */
-class UnexpectedAppendOffsetException(val message: String,
-                                      val firstOffset: Long,
-                                      val lastOffset: Long) extends RuntimeException(message) {
+public class OffsetsOutOfOrderException extends RuntimeException {
```

Comment:
Could you please remove `OffsetsOutOfOrderException.scala`?

Reply:
Done, thanks!